lib/scanner/index.js

```diff
@@ -100,13 +100,17 @@ class Scanner extends Emitter {
    *
    * We don't fire the callback until all interactions
    * (touches/animations) are complete to prevent
-   * re-rendering and droppping frames.
+   * re-rendering and dropping frames.
    */
-  onItemFound({ url, distance }) {
+  onItemFound({ url, distance, latitude, longitude }) {
```

Comment:
I don't have any opinions. Tell me which syntax to use for non existing properties and I'll do it.
On this topic, that'd be nice to have a minimal coding standard guide so that we can list this type of details.